devel/stgit/Portfile

```diff
@@ -27,45 +29,37 @@ long_description    StGIT is a Python application providing similar \
                     the StGIT commands.
 
 homepage            http://procode.org/stgit/
-master_sites        http://repo.or.cz/stgit.git/snapshot/
 
-checksums           rmd160  60261f23324c1bb6c5165cc23a9280850ef23c34 \
-                    sha256  467e2271fb48d73733aa756ed68e53525bce7c4d52a12901f3fca310ba17e310
+set python_default_version 38
 
-distname            48e5cef14cea5c810833d119900cd484c2a6ca85
-worksrcdir          stgit-48e5cef
+checksums           rmd160 166d780c5f57b1fd0dec5338a17234c9c2e43a37 \
+                    sha256 17c2b2e02341468f4d5f8d4d79c36f7fdb7febe18177277ca472502f673c50fd \
+                    size   314721
 
 depends_run         port:git
+depends_lib         port:python${python.version}
 
-python.default_version  27
-
-patch.pre_args      -p1
-
-patchfiles          gerrit-integration.diff \
-                    template.diff
-
-post-patch {
-    reinplace "s|@@PREFIX@@|${prefix}|g"  ${worksrcpath}/stgit/templates.py
-}
-
-set stgdocs         "${worksrcpath}/Documentation/\[a-z\]*.txt"
+set stgdocs         "${worksrcpath}/Documentation/stg.txt"
 set stgman          ""
 
 post-build {
-    system -W ${worksrcpath} "PYTHON=${prefix}/bin/python2.7 make all"
+    system -W ${worksrcpath} "make all"
 }
 
 post-destroot {
     move ${destroot}${python.prefix}/share/stgit/ \
         ${destroot}${prefix}/share/
 
+    move ${destroot}${prefix}/Library/Frameworks/Python.framework/Versions/${python.branch}/bin/stg \
```

Comment:
As demonstrated in the preceding command, you would use `${python.prefix}` here instead of `${prefix}/Library/Frameworks/Python.framework/Versions/${python.branch}`, but in fact, this whole line should be removed, because this shouldn't be moved. Let the python portgroup create the symlink in ${destroot}${prefix}/bin for you instead; it will do so automatically.